src/main/java/com/google/api/codegen/viewmodel/OutputView.java

```diff
@@ -135,12 +135,34 @@ public static Builder newBuilder() {
     }
   }
 
+  @AutoValue
+  abstract class PrintArgView {
+
+    public abstract String arg();
```

Comment:
This name isn't very explanatory.  Could you change it to something like `name` or `formattedName`?

(in a different file above, you  have `arg.arg()` referring to this function, which is very cryptic)

Reply:
That sounds good.